build.gradle.kts

```diff
@@ -188,3 +189,35 @@ subprojects {
     }
   }
 }
+
+allprojects {
```

Comment:
should this be combined with the above `allprojects` block?

Reply:
I believe that having lots of independent `allProjects` blocks is best for keeping logic self-contained